recipe/meta.yaml

```
@@ -20,24 +20,24 @@ requirements:
     - python
   run:
     - ansimarkup >=1.0.0
-    - click >=7.0
     - colorama >=0.4
+    - click >=7.0
     - cryptography
-    - graphene ==2.1.6
-    - jinja2 >=2.10.1
-    - markupsafe >=1.1
-    - metomi-isodatetime ==1!2.0.2
+    - graphene >=2.1,<3
+    - jinja2 ==2.11.0,<2.12
+    - metomi-isodatetime ==1!2.0.1
     - protobuf ==3.12.1
     - pyasn1
     - python
+    - pyzmq >=18.1.0,<19
```

cylc-flow has `pyzmq==18.1.*` so I think we need:

```suggestion
    - pyzmq >=18.1.0,<18.2.0
```